src/main/webapp/app/components/panels/CompanionDiagnosticDevicePanel.tsx

```diff
@@ -29,11 +30,11 @@ export const defaultAdditional = {
 };
 
 const CompanionDiagnosticDevicePanel: React.FunctionComponent<StoreProps> = props => {
-  const [selectedGeneId, setSelectedGeneId] = useState(null);
-  const [alterationValue, onAlterationChange] = useState(null);
-  const [cancerTypeValue, onCancerTypeChange] = useState(null);
-  const [drugValue, onDrugChange] = useState(null);
-  const [fdaSubmissionValue, onFdaSubmissionChange] = useState(null);
+  const [selectedGeneId, setSelectedGeneId] = useState<string | null>(null);
+  const [alterationValue, onAlterationChange] = useState<readonly AlterationSelectOption[] | null>(null);
+  const [cancerTypeValue, onCancerTypeChange] = useState<CancerTypeSelectOption | null>(null);
+  const [drugValue, onDrugChange] = useState<readonly DrugSelectOption[]>([]);
+  const [fdaSubmissionValue, onFdaSubmissionChange] = useState<readonly FdaSubmissionSelectOption[]>([]);
```

Comment:
I got more annoyed with fixing broken tests so some of the code so I just left it alone. It's probably not a big deal to change, but I didn't feel like taking a minor risk. 😆

Reply:
I'm keeping `const [cancerTypeValue, onCancerTypeChange] = useState<CancerTypeSelectOption | null>(null);` the same because `onChange` in `CancerTypeSelect` passes a `null`.

`<CancerTypeSelect onChange={onCancerTypeChange} />`